### ISSUE TITLE
Collections count is incorrectly set to Makes count, often making collections inaccessible

### ISSUE THREAD

**Issue by DrLex0** [DrLex0/ThingiverseIssues#9]
On Thing pages, the collections tab shows zero or a very low number even when the Thing has many collections. If it shows zero, the user cannot reach the Collections tab by clicking this button because it is greyed out. In that case, the only way to get to the collections page is to manually append “`/collections`” to the URL.

**Comment by w3ace** [comment]
This appears to be specifically when items don't have any 'makes' I believe the collection link appears after a make is added.

**Comment by DrLex0** [comment]
You're right, upon closer investigation it seems that the number of collections shown below the gallery is always the same as the number of makes. This is just a bad case of source code copy-paste…
